ui/src/main/webapp/app/toolbar/import-export-dialog/import-export-dialog.html

```diff
@@ -0,0 +1,18 @@
+<md-dialog-content>
+    <div class="md-dialog-content">
+        <h2 class="md-headline">Import or Export results</h2>
+
+
+        Import your results from a Gaffer JSON file.
+        <input type="file" id="import-results-file" name="file" ng-model="importFilename"/>
+        <div style="display:flex; justify-content:flex-end; width:100%; padding:0;">
+            <md-button class="md-raised" ng-click="import()">Import</md-button>
+        </div>
+
+        Export your results to a Gaffer JSON file.
+        <div style="display:flex; justify-content:flex-end; width:100%; padding:0;">
```

Comment:
Could you move this styling into a css class

Reply:
Done - I've reused the CSS and tagging that is already available - also enhanced the design of the modal slightly that is more in line with the rest of the app design (in terms of formatting and colouring)

Reply:
Hi @d47853 - is this CSS change ok?